### ISSUE TITLE
Chat history in logs

### ISSUE THREAD

**Issue by xuhcc** [notsecure/uTox#1073]
When starting, uTox writes chat history to the system log (systemd journal in my case):
```
utox.desktop[906]: loaded backlog: 5: message1
utox.desktop[906]: loaded backlog: 5: message2
...
```
Can it be turned off?

**Comment by GrayHatter** [comment]
@xuhcc what distro?

**Comment by xuhcc** [comment]
@GrayHatter Arch Linux.

**Comment by GrayHatter** [comment]
how are you launching utox?

**Comment by xuhcc** [comment]
Usually from the Gnome menu, but when I launch `utox` from command line, chat history also appears in journal.

**Comment by GrayHatter** [comment]
seems like a misconfiguration, also being on Arch, I don't have any of
those messages.

uTox does use stdout, so I'd ask on the bbs.archlinux or #archlinux on
freenode. If they insist it's uTox, post here or join #tox on freenode, and
I'll see if I can't help you figure it out.

**Comment by irungentoo** [comment]
uTox prints a bunch of debug info to stdout

**Comment by xuhcc** [comment]
@irungentoo Yes, history is displayed among the other debug messages.

**Comment by ElLamparto** [comment]
I've noticed the same thing in Debian Jessie. The whole chat history in the system log.

**Comment by GrayHatter** [comment]
fixed in 5a5132ff1
